source/internal/_rebuild.js

```diff
@@ -0,0 +1,11 @@
+import _fromPairs from './_fromPairs.js';
+import _toPairs from './_toPairs.js';
+import _chain from './_chain.js';
+
+var rebuild = function(convert, obj) {
+  return _fromPairs(_chain(
```

Comment:
I feel like we should go with just `map` to start with here rather than forcing users to wrap all their return values in an array for the most common use case.

Reply:
Well, that's the one I didn't include from earlier discussions.  I certainly don't mind including it.  In my earlier discussion, I called it `remap`, which still sounds all right to me.  But I would like to also have the flexibility that `rebuild` offers.  Whether they nest like I've done (with `remap` between `rebuild` and `mapKeys`) is an interesting question.

Reply:
ok let's just keep what you have here then, no need for both right now